ui/middleware/redirect-to-https.js

```diff
@@ -0,0 +1,13 @@
+export default function ({ req, redirect }) {
+  if (process.server) {
+const trustedProxies = ['127.0.0.1', '10.0.0.1']; // Add your trusted proxy IPs
```

Comment:
@crissthiandi did you review that those are our trusted proxies?

Reply:
ya quite la funcionalidad de proxies porque no me funcionaba ni hacia sentido la logica que propuso el bot

Reply:
@crissthiandi si hace sentido, solo que te dio unos ejemplos de los trusted proxies, era revisar cuales son los nuestros, pero esta bien no implementarlo atm